tests/unit/common/components/input.spec.js

```diff
@@ -0,0 +1,144 @@
+import { mount } from "@vue/test-utils"
+import Input from "@/common/components/Input"
+// import Button from "@/common/components/Button"
```

Comment:
how about this one?

Reply:
my mad miss, i forgot to save after delete this line :( 
updated: this line deleted on next commit